client/gutenberg/extensions/slideshow/settings.js

```diff
@@ -0,0 +1,71 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+
+export const settings = {
+	name: 'slideshow',
+	prefix: 'jetpack',
+	title: __( 'Slideshow' ),
+	category: 'jetpack',
+	keywords: [ __( 'slideshow' ), __( 'image' ) ],
```

Comment:
I think adding a slideshow is redundant since slideshow is in the name already

Reply:
Good point. Removed in https://github.com/Automattic/wp-calypso/pull/29419/commits/ef53a1dcc0edc3be0a76d771099bf8d401e8516a

Reply:
Here are some keyword ideas from the tiled gallery: https://github.com/Automattic/wp-calypso/issues/29780